src/components/FeedbackButton/FeedbackButton.spec.tsx

```diff
@@ -0,0 +1,369 @@
+import {
+  act,
+  cleanup,
+  fireEvent,
+  render,
+  waitForElementToBeRemoved,
+  within,
+} from '@testing-library/react';
+import React from 'react';
+import sinon from 'sinon';
+import FeedbackButton from './FeedbackButton';
+
+describe('<FeedbackButton />', () => {
+  afterEach(() => {
+    sinon.restore();
+    cleanup();
+  });
+
+  it('submits feedback via doSubmit callback with all default properties', async () => {
+    const doSubmit = sinon.stub();
```

Comment:
In each test, this can be replaced with
```javascript
const doSubmit = jest.fn()
```
...and then with the equivalent jest matchers at the end.

No more sinon required!

Reply:
oops yea missed one.